### ISSUE TITLE
Mention `additionalIdentifier` in the identifier reference text

### ISSUE THREAD

**Issue by JachymHercher** [open-contracting/standard#1523]
(Linked to https://github.com/open-contracting/standard/issues/452 (but more specific) and to https://github.com/open-contracting/standard/issues/1215 (but concerning different fields).)

https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#identifier currently states that 

> If a contracting process represents a contract arranged by the department or branch of a larger organization, the legal entity (usually the registered organization) should be described in the [identifier](https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#identifier) section, with details of the branch or department given in the name, [address](https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#address) and [contact point](https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#contactpoint) as relevant.

1) I'm not sure we need this section? 

    (More generally, I'm not sure whether we need (the entire) https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference or possibly some parts of it (e.g. https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#building-block-reference). Aren't field descriptions sufficient for creating the normative obligations? Also, I don't know whether this additional section adds some "overview" value and whether, if it's mainly an "overview", it shouldn't rather be part of guidance?)

2) If we want to have it, I would suggest the following addition on the last line, so that we are in line with https://standard.open-contracting.org/latest/en/guidance/map/organizational_units/ . 

    > If a contracting process represents a contract arranged by the department or branch of a larger organization, the legal entity (usually the registered organization) should be described in the [`identifier`](https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#identifier) section, with details of the branch or department given in the `name`, **`additionalIdentifiers`**,[`address`](https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#address) and [`contact point`](https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#contactpoint) as relevant.

**Comment by jpmckinney** [comment]
Instead of adding more field names, we can maybe replace with:

> When describing an organizational unit (for example, a department or branch of an organization), the `identifier` field should identify the main organization. The other fields should describe the organizational unit.

---

> More generally, I'm not sure whether we need (the entire) https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference

There are a lot of normative statements on this page that do not occur elsewhere (I think "Publishers must not edit a release after publication" is one example). For some objects, there are paragraphs preceding the list of fields that adds a bit more explanatory content that is not specific to any given example, e.g. https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#document

That said, for the most part, the reference is the schema in a tabular format. Most people prefer to consult the schema in an rendered format, rather than the original JSON. We also offer an interactive browser, for exploring hierarchically.

We tend to follow https://documentation.divio.com's division of documentation into tutorials, how-to guides, reference, and explanation. In OCDS, these are:

* primer: explanation
* guidance: tutorials (the 2nd-level pages) and how-to guides (the 3rd-level examples)
* reference: reference

In other words, we can't put reference into guidance, because that's a confusion of purposes.

**Comment by duncandewhurst** [comment]
> Instead of adding more field names, we can maybe replace with:
> 
> > When describing an organizational unit (for example, a department or branch of an organization), the `identifier` field should identify the main organization. The other fields should describe the organizational unit.

That sounds good to me so I've prepared a PR. I also added a link to the organizational units guidance.
